src/OrchardCore.Modules/OrchardCore.Users/GraphQL/CurrentUserQuery.cs

```diff
@@ -21,45 +17,23 @@ namespace OrchardCore.Users.GraphQL;
 internal sealed class CurrentUserQuery : ISchemaBuilder
 {
     private readonly IHttpContextAccessor _httpContextAccessor;
-    private readonly IOptions<GraphQLContentOptions> _contentOptionsAccessor;
     private readonly IStringLocalizer S;
 
     public CurrentUserQuery(
         IHttpContextAccessor httpContextAccessor,
-        IOptions<GraphQLContentOptions> contentOptionsAccessor,
         IStringLocalizer<CurrentUserQuery> localizer)
     {
         _httpContextAccessor = httpContextAccessor;
-        _contentOptionsAccessor = contentOptionsAccessor;
         S = localizer;
     }
 
-    public async Task BuildAsync(ISchema schema)
+    public Task BuildAsync(ISchema schema)
     {
-        // Build a user type that includes all custom user settings.
-        var serviceProvider = _httpContextAccessor.HttpContext.RequestServices;
-        var contentDefinitionManager = serviceProvider.GetRequiredService<IContentDefinitionManager>();
-        var contentTypes = await contentDefinitionManager.ListTypeDefinitionsAsync();
-
-        var userType = serviceProvider.GetRequiredService<UserType>();
-
-        // Note: The content types are already added to GraphQL by the ContentTypeQuery. Just add them to the user type.
-        foreach (var typeDefinition in contentTypes.Where(t => t.StereotypeEquals("CustomUserSettings")))
-        {
-            // Skip hidden types
-            if (_contentOptionsAccessor.Value.ShouldHide(typeDefinition))
-            {
-                continue;
-            }
-
-            userType.AddField(schema, typeDefinition);
-        }
-
         var currentUserField = new FieldType
         {
             Name = "me",
             Description = S["Gets the currently authenticated user."],
-            ResolvedType = userType,
+            Type = typeof(UserType),
```

Comment:
Where the `UserType` is defined?

Reply:
https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Users/GraphQL/UserType.cs